tests/test_groups.py

```diff
@@ -42,26 +66,24 @@ def test_create_deregister(self):
         self.assert_group_not_exists(outer_name)
         self.assert_group_not_exists(inner_name)
 
-    def assert_group_exists(self, name):
-        """Asserts that a group exists, and gives it a few tries."""
-        api = client.build(rest_api.GroupsApi)
-        for _ in range(TRIES):
-            try:
-                api.get_group_contents(group_namespace=self.namespace, group_name=name)
-                return
-            except rest_api.ApiException:
-                pass  # try again
-            time.sleep(1)
-        self.fail(f"group {self.namespace}/{name} does not exist")
+    def test_update_group_info(self):
+        group_name = testonly.random_name("test_group_info")
+        group_storage_name = testonly.random_name(group_name)
+        group_storage_path = f"{self.test_path}/{group_storage_name}"
+        group_uri = f"tiledb://{self.namespace}/{group_name}"
+        groups.create(group_name, storage_uri=group_storage_path)
+        self.assert_group_exists(group_name)
 
-    def assert_group_not_exists(self, name):
-        """Asserts that a group does not exist, giving it a few tries to go."""
-        api = client.build(rest_api.GroupsApi)
-        for _ in range(TRIES):
-            try:
-                api.get_group_contents(group_namespace=self.namespace, group_name=name)
-            except rest_api.ApiException as apix:
-                if apix.status in (400, 404):
-                    return
-            time.sleep(1)
-        self.fail(f"group {self.namespace}/{name} still exists")
+        description = "this is a test description"
+        logo = "testLogo"
+        tags = ["tag", "othertag"]
+        groups.update_info(group_uri, description=description, logo=logo, tags=tags)
+        group_info = groups.info(group_uri)
+
+        self.assertEqual(group_info.description, description)
+        self.assertEqual(group_info.logo, logo)
+        self.assertCountEqual(group_info.tags, tags)
+
+        # Cleanup
+        groups.deregister(group_uri)
```

Comment:
I think we should delete not just deregister so we cleanup storage too.

Reply:
I have just copied the "delete"/deregister from the existing test.
It has a TODO about that same issue. Do we have a real delete method?